com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/SSEXMLFile.java

```diff
@@ -82,5 +83,61 @@
 		return _results;
 	}
 
+	@Override
+	public Collection<SearchResult> findLegacyDependency(String groupId, String artifactId) {
```

Comment:
hey Greg,  I found that javax.xml.xpath.XPath can't support org.eclipse.wst.xml.core.internal.provisional.document.IDOMDocument , but eclipse-wtp-xml-search project did add such support for IDOMDocument , but it will be more complex if we add xml-search bundles in blade. What't your opinion ?